modules/ROOT/pages/database-administration/syntax.adoc

```diff
@@ -78,7 +78,9 @@ REQUIRE n.propertyName IS NOT NULL
 [[administration-syntax-database-management]]
 == Database management command syntax
 
-The below table covers both standard and composite databases.
+The following tables cover both standard and composite databases.
```

Comment:
You might want to clarify that it is the following two (related to SHOW) which cover both, as the third and fourth table after this sentence (related to CREATE) is not for both

Reply:
same here about it being the tables as a collective that covers both, not specifically each individual one

Reply:
but maybe something as easy as
```suggestion
The following tables cover standard and composite databases.
```
might do the trick?

Reply:
(I also think at least DROP covers both, and I guess we'll see what we do for alter composite in https://github.com/neo4j/docs-operations/pull/2153, if we split it or just use the same 🤷)

Reply:
How about:
```suggestion
The database management commands are used to manage standard or composite databases.
```